packages/type/scss/index.scss

```diff
@@ -5,7 +5,5 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import 'font-family';
```

Comment:
Can you make these changes in `type.scss` as well? We support two Sass entrypoint files per package.

Reply:
Just noticed that `styles.scss` imports `scale` but `index.scss` and `type.scss` are also importing it. Should I also remove that?

Reply:
Sounds good to me!